webpack.config.js

```diff
@@ -0,0 +1,60 @@
+const path = require('path');
+const webpack = require('webpack');
+const ExtractTextPlugin = require('extract-text-webpack-plugin');
+const packageData = require('./package.json');
+
+
+module.exports = {
+  entry: {
+    admin: ['./admin/assets/admin.js', './admin/assets/admin.css'],
+    vendor: ['semantic-ui-css', 'semantic-ui-css/semantic.css', 'jquery']
+  },
+  output: {
+    filename: '[name].bundle.min.js',
+    path: path.resolve(__dirname, 'dist/admin/assets')
+  },
+  devtool: 'source-map',
```

Comment:
Hey! Good idea! Looks like that is supposed to be faster, too. Currently the file isn't even set to produce source maps in production. Maybe we should, though...